spotbugs-exclude.xml

```diff
@@ -23,6 +23,15 @@
 -->
 <FindBugsFilter>
     <!-- contains specific exclusions of bug patterns -->
+    <Match>
+        <Or>
+            <Class name="org.tweetwallfx.cache.URLContentCacheBase$1" />
+            <Class name="org.tweetwallfx.cache.URLContentCacheBase$2" />
+            <Class name="org.tweetwallfx.config.ConfigurationTest" />
```

Comment:
This entry is not necessary when you remove the throws declaration from the test method. That throws is unnecessary

Reply:
Also something you already did. So should be oky to remove this entry
```suggestion
```